feg/gateway/services/aaa/radius/config.go

```diff
@@ -37,7 +39,7 @@ var defaultConfigs = &mconfig.RadiusConfig{
 func ValidateConfigs(cfg *mconfig.RadiusConfig) *mconfig.RadiusConfig {
 	res := &mconfig.RadiusConfig{}
 	if cfg != nil {
-		*res = *cfg
+		res = proto.Clone(cfg).(*mconfig.RadiusConfig)
```

Comment:
This seems wrong. We are cloning an existing object when we validate the config. 
Please can you try to play with the code to remove that clone function

Reply:
Will look into it.